model/test_utils.go

```diff
@@ -57,8 +57,8 @@ func GenerateTestLog(ctx context.Context, bucket pail.Bucket, size, chunkSize in
 	return chunks, lines, nil
 }
 
-// GenerateSystemMetrics is a convenience function to generate a specified number of
-// 16 byte random system metrics data chunks in the specified bucket.
+// GenerateSystemMetrics is a convenience function to generate a specified
+// number of 32 byte random system metrics data chunks in the specified bucket.
```

Comment:
Thank you, this would be confusing otherwise.

Reply:
Yes! I was stuck for like 2 hours testing cause I though it was 16 bytes and I couldn't figure out why the pagination wasn't working! It was working, I was just off with the size of the chunks...